content/docs/reference-events.md

```diff
@@ -31,44 +31,44 @@ number timeStamp
+> Sejak v0.14, mengembalikan nilai `false` dari _event handler_ tidak akan menghentikan propagasi _event_. Sebagai gantinya, `e.stopPropagation()` atau `e.preventDefault()` perlu dipanggil secara manual.
 
 ### Event Pooling {#event-pooling}
 
-The `SyntheticEvent` is pooled. This means that the `SyntheticEvent` object will be reused and all properties will be nullified after the event callback has been invoked.
-This is for performance reasons.
-As such, you cannot access the event in an asynchronous way.
+`SyntheticEvent` dikumpulkan. Hal ini berarti objek `SyntheticEvent` akan digunakan kembali dan isi objek akan dikosongkan setelah _callback_ untuk _event_ tersebut dipanggil.
+Hal ini dilakukan untuk mengoptimalkan kinerja.
+Dengan demikian, _event_ tersebut tidak dapat diakses secara asinkron.
 
 ```javascript
 function onClick(event) {
-  console.log(event); // => nullified object.
-  console.log(event.type); // => "click"
-  const eventType = event.type; // => "click"
+  console.log(event); // => objek yang dikosongkan.
+  console.log(event.type); // => "klik"
+  const eventType = event.type; // => "klik"
 
   setTimeout(function() {
     console.log(event.type); // => null
-    console.log(eventType); // => "click"
+    console.log(eventType); // => "klik"
   }, 0);
 
-  // Won't work. this.state.clickEvent will only contain null values.
+  //Tidak akan berfungsi. this.state.clickEvent hanya berisi nilai kosong.
   this.setState({clickEvent: event});
 
-  // You can still export event properties.
+  // Anda masih dapat mengekspor atribut event.
   this.setState({eventType: event.type});
 }
 ```
 
-> Note:
+> Catatan:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
+> Jika Anda ingin mengakses isi _event_ secara asinkron, Anda perlu memanggil fungsi `event.persist()` dalam _event_, yang akan mengeluarkan _synthetic event_ dari kumpulan sehingga acuan terhadap _event_ tersebut dapat dipertahakan melalui kode pengguna.
 
 ## Supported Events {#supported-events}
 
-React normalizes events so that they have consistent properties across different browsers.
+React menormalisasikan _event_ agar memiliki isi yang sama dalam seluruh _browser_.
 
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
+_Event handler_ dibawah dipicu oleh _event_ dalam fase _bubbling_. Untuk mendaftarkan sebuah _event handler_ dalam fase _capture_, tambahkan `Capture` pada nama _event_; contohnya, gunakan `onClickCapture` untuk menangani _event_ klik dalam fase _capture_ sebagai ganti `onClick`.
```

Comment:
dibawah -> di bawah

```suggestion
_Event handler_ di bawah dipicu oleh _event_ dalam fase _bubbling_. Untuk mendaftarkan sebuah _event handler_ dalam fase _capture_, tambahkan `Capture` pada nama _event_; contohnya, alih-alih menggunakan `onClick` gunakan `onClickCapture` untuk menangani _event_ klik dalam fase _capture_.
```

What do you think? 😄

Reply:
:+1: , I would probably add a comma though after "alih-alih menggunakan `onClick`". wdyt?

Reply:
Good idea 👍